cram/cram_encode.c

```diff
@@ -1978,12 +2015,14 @@ int cram_encode_container(cram_fd *fd, cram_container *c) {
 
 
     /* Compute MD5s */
+    no_ref = c->no_ref;
```

Comment:
I don't think this line is needed.

Reply:
The function has grown over the years and is overly long, so it really needs a refactor.  However `no_ref` is used a few lines lower down and above it there are several places that modify `c->no_ref`.  It's possible the logic is such that they never have any effect on the cached `no_ref`, but I wouldn't like to say with 100% certainty and for clarity alone it's worth keeping this line IMO.